annotation-file-utilities/src/annotator/find/Insertion.java

```diff
@@ -203,6 +247,50 @@ public static void setAlwaysQualify(Set<String> set) {
     }
 
     /**
+     * Gets the type.  It is assumed that the returned value will be
+     * modified to update the type to be inserted.
+     */
+    public Type getType() {
+        return type;
+    }
+
+    /**
+     * Sets the type.
+     */
+    public void setType(Type type) {
+        this.type = type;
+    }
+
+    public DeclaredType getBaseType() {
```

Comment:
Every method needs Javadoc documentation.  Please write it for this method and any other methods that lack it.  I see some more in this class, and others might exist elsewhere in the pull request.